tests/envs/test_envs.py

```diff
@@ -92,12 +94,13 @@ def test_random_rollout():
 
 def test_env_render_result_is_immutable():
```

Comment:
I know this is an existing text, but can we parametrize it over all envs? I'm not sure which ones support `ansi`, but doing the test for all of them would be nice. (you can probably just explicitly skip envs which don't have `ansi in self.metadata['render_modes']`

Reply:
Yes, should it be done in another PR?

Reply:
I’m looking at upgrading the environment tests, so I can include it in my PR

Reply:
@pseudo-rnd-thoughts it makes more sense, thanks